### DISCUSSION TITLE
是否考虑打个 winget 包？

### DISCUSSION THREAD

**Discussion by HydrogenDeuterium** [NotepadFree/NotepadFree#4] [Ideas]
最近重装系统，大部分软件都直接
`winget install xxx`
了，感觉打一个winget的包挺好的。

好像实际上只要往 winget 的仓库里说明一下到哪里去下这个软件，但是我之前只作为 winget 用户而不是开发者所以不太清楚具体要怎么做。

睡前知乎看到这个项目的（甚至这个贴也是手机打字），火速过来 star，等睡醒了我研究一下怎么整

**Comment by SongZihuan** [comment]
确实是个不错的提议。
